src/app/services/title/title.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
+import { TitleList } from '../../mock/mock-title';
+import { Title } from '../../interfaces/title/title.interface';
+
+@Injectable()
+
+export class TitleService {
```

Comment:
Don't forget to remove TitleService in the end.

Reply:
was applied